Resources/shaders/basic.shader

```diff
@@ -0,0 +1,73 @@
+#shader vertex
```

Comment:
This file could just have a little tidying up:

- new lines between functions
- spaces around operators, e.g. `uv.y*12.345` -> `uv.y * 12.345`
- use `const` wherever possible